stdlib/Test/src/Test.jl

```diff
@@ -89,16 +89,9 @@ struct Pass <: Result
 end
 function Base.show(io::IO, t::Pass)
     printstyled(io, "Test Passed"; bold = true, color=:green)
-    if !(t.orig_expr === nothing)
-        print(io, "\n  Expression: ", t.orig_expr)
-    end
     if t.test_type === :test_throws
         # The correct type of exception was thrown
         print(io, "\n      Thrown: ", typeof(t.value))
-    elseif t.test_type === :test && t.data !== nothing
-        # The test was an expression, so display the term-by-term
-        # evaluated version as well
-        print(io, "\n   Evaluated: ", t.data)
```

Comment:
Why don't we want to display this?
Given someone has created a test result object, why not show its fields?

Reply:
I can see it being argued either way.

One the one hand the information is there so why not display it to give the user more information, but on the other hand if a test has passed the user may not want the details of the passing of that test to fill up the REPL. I typically only see the `Test Passed` output when manually stepping through tests to find the failing ones, and wouldn't be interested in knowing the details of the ones that passed. Normally the tests are in `testsets` and therefore the pass information isn't shown either way.

I kept it to the latter as that's what was originally happening before the changes here, albeit because that information wasn't stored so couldn't be displayed anyway.

I'm leaning towards not displaying it, but its definitely worth a discussion.

Reply:
Fair enough. I am not sure when this is displayed, except if `@test` is used on its out, in the REPL.
 do `@testsets` call display on the tests with in them?